src/Grpc.StatusProto/README.md

```diff
@@ -0,0 +1,256 @@
+﻿# Grpc C# API for error handling with Status.proto
+
+This is a protoype NuGet package providing client and server side support for the
+[gRPC richer error model](https://grpc.io/docs/guides/error/#richer-error-model).
+
+It had dependencies NuGet packages on:
+* `Google.Api.CommonProtos` - to provide the proto implementations used by the richer error model
```

Comment:
as discussed offline, if some of the C# gets added to Google.Api.CommonProtos as extension methods, let's pay close attention to in which namespace these extension methods live, since we don't want to require adding too many "using x.y.z" statements to be able to use the Grpc.StatusProto functionality (in order for the extension methods to become visible).